src/components/Footer/footer.js

```diff
@@ -1,41 +1,61 @@
 import { BrowserRouter as Router, NavLink } from "react-router-dom";
 import styles from "./footer.module.scss";
+import logo from "../../assets/image/footer/Layer 2.png";
 
 export const Footer = () => {
   return (
     <footer className={styles.footer}>
-      <img className={styles.logo} alt="logo" />
+      <div className={styles.logo_column}>
```

Comment:
This column's content starts at a other height than the content in the other two.